<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Condensed</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Locate Me Univalle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widgetHeader" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>71</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>57</red>
          <green>77</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>57</red>
          <green>77</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>158</green>
          <blue>158</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>57</red>
          <green>77</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>57</red>
          <green>77</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSubtitle">
        <property name="text">
         <string>Tree Algoritm Search</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelTitle">
        <property name="styleSheet">
         <string notr="true">font: 30pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Locate Me Univalle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelDelectedSearch">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>16</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Select search from menu</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetBody" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>981</width>
      <height>461</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelRobotStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Robot Status</string>
     </property>
    </widget>
    <widget class="Line" name="lineRobotStatus">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>10</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="lineActualNode">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>120</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelActualNode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Actual Node</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDataStatus">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>0</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data Status</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditDataStatus">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>251</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelEnvironment">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Environmet</string>
     </property>
    </widget>
    <widget class="Line" name="lineEnvironmet">
     <property name="geometry">
      <rect>
       <x>367</x>
       <y>10</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>331</width>
       <height>227</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelState">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>state</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditState"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelParent">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>parent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditParent"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelAction">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Action</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditAction"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelCost">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cost</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditCost"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDepth">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditDepth"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelH">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>h(n)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditH"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelG">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>g(n)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditG"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>331</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelMessage">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelBattery">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Battery</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditBattery"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditMessage"/>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="lineRobotStatus_2">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>400</y>
       <width>961</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelRobotStatus_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMoveRobot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Move robot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLookGraph">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Look Graph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNextStep">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>420</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Next Step</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFastSearch">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>420</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fast search</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetEnvironment">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>331</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>246</red>
          <green>244</green>
          <blue>242</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QMenu" name="actionUniformCost">
     <property name="title">
      <string>Uniform Cost </string>
     </property>
    </widget>
    <widget class="QMenu" name="actionA_2">
     <property name="title">
      <string>A*</string>
     </property>
    </widget>
    <addaction name="actionUniformCost"/>
    <addaction name="actionA_2"/>
   </widget>
   <widget class="QMenu" name="menuEnvironment">
    <property name="title">
     <string>Environment</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionLook_Actual"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSearch"/>
   <addaction name="menuEnvironment"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionLook_Actual">
   <property name="text">
    <string>Look Actual</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUniform_Cost">
   <property name="text">
    <string>Uniform Cost</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>Uniform Cost</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>A* </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menubar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
